internal/uvm/create_wcow.go

```diff
@@ -122,6 +123,16 @@ func prepareConfigDoc(ctx context.Context, uvm *UtilityVM, opts *OptionsWCOW, uv
 		}
 	}
 
+	processor := &hcsschema.Processor2{
+		Count:  uvm.processorCount,
+		Limit:  opts.ProcessorLimit,
+		Weight: opts.ProcessorWeight,
+	}
+	// We can set a cpu group for the VM at creation time in recent builds.
+	if opts.CPUGroupID != "" && osversion.Build() >= 20124 {
```

Comment:
Seems like we should return an error if a CPU group is requested but we don't support it on this build.

Reply:
good call